nerfstudio/models/splatfacto.py

```diff
@@ -156,6 +156,22 @@ class SplatfactoModelConfig(ModelConfig):
     """Shape of the bilateral grid (X, Y, W)"""
     color_corrected_metrics: bool = False
     """If True, apply color correction to the rendered images before computing the metrics."""
+    strategy: Literal["default", "mcmc"] = "default"
+    """The default strategy will be used if strategy is not specified. Other strategies, e.g. mcmc, can be used."""
+    cap_max: int = 1_000_000
+    """Maximum number of GSs. Default to 1_000_000."""
+    noise_lr: float = 5e5
+    """MCMC samping noise learning rate. Default to 5e5."""
+    min_opacity: float = 0.005
+    """GSs with opacity below this value will be pruned. Default to 0.005."""
+    verbose: bool = False
```

Comment:
i think we can delete this and default to false for the sake of simplicity (since it only affects the strategies)